pages/common/gitlab-runner.md

```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
+> More information: <https://docs.gitlab.com/runner/>.
+
+- Register a runner:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}}`
+
+- Register a runner with a specific executor:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}} --executor {{docker}}`
+
+- Unregister a runner:
+
+`sudo gitlab-runner unregister --name {{runner_name}}`
+
+- Get the status of the runner service:
+
+`sudo gitlab-runner status`
+
+- Restart the runner service:
+
+`sudo gitlab-runner restart`
+
+- Verify registered runners are running:
```

Comment:
The doc says this is the output:
```
Verifying runner... is alive                        runner=fee9938e
```
Do the command say if the runner is just connected or if running?

Reply:
I changed it to use the same wording as the doc.